test/unit/modules/app.js

```diff
@@ -1,37 +1,28 @@
 'use strict';
 
 var chai = require('chai');
+var child_process = require('child_process');
```

Comment:
These two requires are not needed. They are not even presented in the `package.json`. Please, remove them.

Reply:
Ok, will remove it. `child_process` is Node.js library and doesn't have to be included in `package.json`